test/interface.jl

```diff
@@ -1,7 +1,7 @@
 import Reinforce: action, actions, finished, ismdp, maxsteps, reset!, reward, state, step!
 
 function test_ep_iteration()
-  info("interface::iteration::maxsteps")
+  @info "interface::iteration::maxsteps"
```

Comment:
I think this `@info` will break the 0.6 support.
(But I'm fine about just dropping 0.6 support.)
Which will you vote fro? dropping or not dropping?

Reply:
I actually added using `Compat: @info` [here in runtests.jl](https://github.com/JuliaML/Reinforce.jl/blob/9a08929bbf62b3c429be2a83e20311ebd24b51d2/test/runtests.jl#L2), so that 0.6 tests pass, but anyway, I think we should drop 0.6 for future changes.  I doubt many people, if anyone, are relying on this to work for 0.6.

Reply:
My OpenAIGym PRs also work on 0.6 btw

Reply:
oh okay

Reply:
Oh actually OpenAIGym.jl/pull/10 works, but the others don't on 0.6

Reply:
So, you mean all the PRs are based on 1.0, right?

Reply:
Yeah, but it's  prob a minor change to get them working for 0.6, OTOH 0.6 is old news 😄

Reply:
well, I will vote for dropping it.